FAQ.md

```diff
@@ -0,0 +1,82 @@
+# FAQ
+
+## How do I run the test suite?
+
+`mvn test` will run unit tests while `mvn verify` will include integration tests and benchmarks.
+
+See [the testing documentation](/TESTING.md) for additional notes about running the test suite.
+
+## How do Accounts/Usernames/Subusers differ?
+
+Joyent's SmartDataCenter account implementation is such that you can have a
+subuser as a dependency upon a user. This is part of SmartDataCenter's [RBAC
+implementation](https://docs.joyent.com/public-cloud/rbac/users). A subuser
+is a user with a unique username that is joined with the account holder's
+username. Typically, this is in the format of "user/subuser".
+
+Within the Java Manta library, we refer to the account name as the entire
+string used to login - "user/subuser". When we use the term user it is in
+reference to the "user" portion of the account name and when we use the term
+subuser, it is in reference to the subuser portion of the account name.
+
+The notable exception is that in the configuration passed into the library,
+we have continued to use the terminology *Manta user* to refer to the
+account name because of historic compatibility concerns.
+
+## Does Triton Object Storage (Manta) support Multipart Uploads?
```

Comment:
This is the first time we're seeing that Triton Object Storage is Manta. Note for us, we need to think of a better way to make this transition.

Reply:
Also, "multipart uploads" is not a proper noun, therefore it should not be capitalized.